### ISSUE TITLE
Block this autistic Google gaystore popup.

### ISSUE THREAD

**Issue by Vader77Darth** [rhunk/SnapEnhance#1173]
### Description

This stupid popup keeps appearing like every 10 seconds and it is enfuriating. Please add an option to
![Screenshot_2024-07-23-07-58-47-592_com google android packageinstaller](https://github.com/user-attachments/assets/3d53d873-5c44-417c-be3b-591e7015bcca)
 block it.

### Agreement

- [X] There is no issue already suggesting my feature.

**Comment by aayushrautela** [comment]
Just remove/disable 'Getapps'

**Comment by Vader77Darth** [comment]
> Just remove/disable 'Getapps' or don't auto update Snapchat

No. I want to use getapps to auto update Snapchat. I should not be forced into the evil of the Google Gaystore.

**Comment by Vader77Darth** [comment]
> I can't do anything about it it's not part of the Snapchat app

Rip. I guess it is another one of Google's inhumane practices....😤

**Comment by aayushrautela** [comment]
> > Just remove/disable 'Getapps' or don't auto update Snapchat
> 
> No. I want to use getapps to auto update Snapchat. I should not be forced into the evil of the Google Gaystore.

Getapps is from Xiaomi. It's 10x more sketchy than play store. If you are rooted, you can detach Snapchat from play store. Regardless, it's not on a SnapEnhance issue.

**Comment by Vader77Darth** [comment]
> > > Just remove/disable 'Getapps' or don't auto update Snapchat
> > 
> > 
> > No. I want to use getapps to auto update Snapchat. I should not be forced into the evil of the Google Gaystore.
> 
> Getapps is from Xiaomi. It's 10x more sketchy than play store. If you are rooted, you can detach Snapchat from play store. Regardless, it's not on a SnapEnhance issue.

It is not a snapenhance issue. That's why this is closed issue. Found out it's actually caused by the package installer (which is still developed by Gaygle). Your claim about the playstore being 10x less sketchy is objectively wrong considering the crimes against humanity Gaygle (developers of the Gaystore) has commited.

**Comment by StupidRepo** [comment]
We never figured out _why_ you needed to call so urgently anyway...

**Comment by Vader77Darth** [comment]
> We never figured out _why_ you needed to call so urgently anyway...

My Netflix subscription ran out. I have a deadly health condition which means that I cannot go for more than 4 hours without watching the 2022 blockbuster film Morbius. I needed her to desperately renew the subscription; my life was at risk. (real)

**Comment by Vader77Darth** [comment]
I found out how to fix it: tap "update anyway."

**Comment by StupidRepo** [comment]
> > We never figured out _why_ you needed to call so urgently anyway...
> 
> My Netflix subscription ran out. I have a deadly health condition which means that I cannot go for more than 4 hours without watching the 2022 blockbuster film Morbius. I needed her to desperately renew the subscription; my life was at risk. (real)

Ahhh, I see...

**Comment by Vader77Darth** [comment]
> I found out how to fix it: tap "update anyway."

I can confirm this does not work anymore

**Comment by aayushrautela** [comment]
Just detach it from play store. What's even all the fuzz about.

**Comment by StupidRepo** [comment]
Real.

**Comment by Vader77Darth** [comment]
> Just detach it from play store. What's even all the fuzz about.

How?

**Comment by StupidRepo** [comment]
With @j-hc's [zygisk detach](https://github.com/j-hc/zygisk-detach) Magisk module

**Comment by Vader77Darth** [comment]
> With @j-hc's [zygisk detach Magisk module](https://github.com/j-hc/zygisk-detach).

Thanks. I will try that.

**Comment by StupidRepo** [comment]
No problem :)
You'll also need a way to run `su -c detach` so I recommend getting Termux if you don't already have it 👍🏽

**Comment by Vader77Darth** [comment]
> > With @j-hc's [zygisk detach Magisk module](https://github.com/j-hc/zygisk-detach).
> 
> Thanks. I will try that.

Yes. This works.

**Comment by StupidRepo** [comment]
> > > With @j-hc's [zygisk detach Magisk module](https://github.com/j-hc/zygisk-detach).
> > 
> > 
> > Thanks. I will try that.
> 
> Yes. This works.

Happy to help :)
